spec/index.html

```diff
@@ -2674,8 +2674,12 @@ <h2>Serialization</h2>
 <section id="privacy-considerations" class="informative">
   <h2>Privacy Considerations</h2>
 
-  <section>
-    <h3>Selective Disclosure Schemes</h3>
+  <p>Privacy considerations here are primarily worth discussing when the canonicalization scheme is used for
+    privacy-respecting signed RDF dataset and are likely acceptable for other use cases. One of the former examples is a
+    verifiable credential with selective disclosure.</p>
```

Comment:
This probably needs re-wording. I'll come up with a suggestion at a later point.

Reply:
Thank you for your re-wording! I applied your suggestion.